lan867x/README.md

```diff
@@ -26,23 +26,34 @@ esp_eth_phy_t *phy = esp_eth_phy_new_lan867x(&phy_config);
-| LAN867X_ETH_CMD_G_EN_PLCA          | bool* enable     | Write PLCA status (true if enabled) to the location via pointer                                           |
-| LAN867X_ETH_CMD_S_PLCA_NCNT        | uint8_t* count   | Set node count to the value passed through the pointer                                                    |
-| LAN867X_ETH_CMD_G_PLCA_NCNT        | uint8_t* count   | Write node count configured in the PLCA to the location via pointer                                       |
-| LAN867X_ETH_CMD_S_PLCA_ID          | uint8_t* id      | Set ID to the value passed through the pointer                                                            |
-| LAN867X_ETH_CMD_G_PLCA_ID          | uint8_t* id      | Write ID configured in the PLCA to the location via pointer                                               |
-| LAN867X_ETH_CMD_ADD_TX_OPPORTUNITY | uint8_t* node_id | Add additional transmit opportunity for chosen node                                                       |
-| LAN867X_ETH_CMD_RM_TX_OPPORTUNITY  | uint8_t* node_id | Remove additional transmit opportunity for chosen node                                                    |
-| LAN867X_ETH_CMD_S_MAX_BURST_COUNT  | uint8_t* maxcnt  | Set max count of additonal packets transmitted during one frame, or 0 to disable PLCA burst               |
-| LAN867X_ETH_CMD_G_MAX_BURST_COUNT  | uint8_t* maxcnt  | Write max count of additonal packets transmitted during one frame to the location via pointer             |
-| LAN867X_ETH_CMD_S_BURST_TIMER      | uint8_t* time    | Set time during which additional packets can be sent in incriments of 100ns                               |
-| LAN867X_ETH_CMD_G_BURST_TIMER      | uint8_t* time    | Write time during which additional packets can be sent in increments of 100ns to the location via pointer |
-| LAN768X_ETH_CMD_PLCA_RST           |                  | Perform reset of the PLCA                                                                                 |
-
-and you are ready to use the Ethernet driver as you are used to. For more information of how to use ESP-IDF Ethernet driver, visit [ESP-IDF Programming Guide](https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-reference/network/esp_eth.html).
+In most cases you can rely on CSMA/CD, which is enabled by default, to avoid collisions when connecting multiple nodes. Its benefits are that it requires zero configuration to work, and does a good job of coordinating when data is sent, however, due to its nature, the delays between transmissions are random, and in timing-critical applications, you are better with deterministic PLCA. 
+
+PLCA requires setting at least PLCA ID, node count, and selecting one node as a coordinator but ensures precise and repeatable time between transmissions, and due to the flexibility of the transmit opportunity timer, burst mode and additional transmit opportunities you can fine-tune it to your needs.
```

Comment:
We could add links to both CSMA/CD and PLCA

Reply:
I think providing link (or other type of reference) to LAN867x how to configure PLCA will be sufficient. No link needed for CSMA/CD since there is no configuration required and it's kind of old approach so there is bunch of information about it on the internet.